Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/EditorComponents/ActionPageComponents/Action.js

```diff
@@ -30,7 +30,7 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
-export default function FinalAction(props) {
+export default function Action(props) {
```

Comment:
Was this change a remnant of old namings? or is this something we should be concerned about?

Reply:
yea, should be called Action component, shouldn't affect anything